src/module-analyzer.js

```diff
@@ -3,6 +3,7 @@ import {Container} from 'aurelia-dependency-injection';
 import {TemplateRegistryEntry} from 'aurelia-loader';
 import {ValueConverterResource} from 'aurelia-binding';
 import {BindingBehaviorResource} from 'aurelia-binding';
+import {ViewEngineHooksResource} from 'aurelia-binding';
```

Comment:
I thought about it, but I don't do a lot of contributing and I wanted to keep with the current conventions / style.